StalledRetry/StalledRequestHandler.js

```diff
@@ -68,9 +71,11 @@ class StalledRequestHandler {
 
     _waitForCompletion(cb) {
         async.whilst(
-            () => (!this.killed && this.isInProgress()),
-            done => setTimeout(done, 1000),
-            cb,
+            done => {
+                done(null, !this.killed && this.isInProgress());
+            },
+            next => setTimeout(next, 1000),
```

Comment:
just use async and keep code leaner:

```suggestion
            async () => (!this.killed && this.isInProgress()),
            done => setTimeout(done, 1000),
```